tests/acceptance/features/webUIRenameFiles/renameFiles.feature

```diff
@@ -102,6 +102,7 @@ Feature: rename files
     Then file "loremy.tad" should be listed on the webUI
 
 
+  @notToImplementOnOCIS
```

Comment:
No need to add a line here - there are already 2 blank lines. One of then is waiting for tags.